### DISCUSSION TITLE
[FEATURE] 如何支持sqlite3数据库？

### DISCUSSION THREAD

**Discussion by feipinxiang** [1Panel-dev/1Panel#5186] [Q&A]
### 1Panel 版本

v1.10.9-lts

### 请描述您的需求或者改进建议

不会docker，想可视化处理。
需求：用php做个页面满足客户的一个小需求，需求太简单想省资源直接用sqlite3
1. 怎么在openresty下的php站点内使用sqlite3? （装了个应用商店里的adminer也没能成功的连接sqlite3数据库文件）
2. 能否在“数据库”管理页面内查看sqlite3？

**Comment by wanghe-fit2cloud** [comment]
1、创建 PHP 运行环境，选择 sqlite3 相关扩展
2、如果是容器化安装 sqlite3，还需要注意两点：
- 如果是容器内访问，要确保 sqlite3 使用 1panel-network 这个默认容器网络
- 如果是要外部 IP 访问，要确保对外暴露 sqlite3 端口
